.buildkite/integration.pipeline.yml

```diff
@@ -5,27 +5,77 @@ env:
   VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
 
 steps:
-  - label: "Integration tests: packaging"
-    key: "package-it"
-    command: ".buildkite/scripts/steps/integration-package.sh"
-    artifact_paths:
-      - build/distributions/**
-    agents:
-      provider: "gcp"
-      machineType: "n1-standard-8"
+  - group: "Integration tests: packaging"
+    key: "int-packaging"
+    steps:
+      - label: "Packaging: Ubuntu x86_64"
+        key: "packaging-ubuntu-x86-64"
+        env:
+          PACKAGES: "tar.gz"
+          PLATFORMS: "linux/amd64"
+        command: ".buildkite/scripts/steps/integration-package.sh"
+        artifact_paths:
+        - build/distributions/**
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
```

Comment:
nit: `n1` is the oldest general purpose machine generation on GCP. With `n2-standard-8` we'd get much better performance with a very small [diff in price](https://cloud.google.com/compute/vm-instance-pricing?hl=en#n2_predefined) (+6$ for continuous month use for n2, not including discounts.)